include/swift/Runtime/Runtime.h

```diff
@@ -0,0 +1,55 @@
+//===--- Runtime.cpp - Runtime functions exposed in Runtime module ---- ===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+// Definitions relating to public runtime module.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef SWIFT_RUNTIME_MANGLING_H
+#define SWIFT_RUNTIME_MANGLING_H
+
+#ifdef __linux__
+#include <sys/types.h>
+#include <sys/wait.h>
+
+#include <signal.h>
+#endif // defined(__linux__)
```

Comment:
Nit: ifdef instead of defined (I do like the defined form myself). Switch either side to be homogeneous.